### DISCUSSION TITLE
Allow blank target on button component

### DISCUSSION THREAD

**Discussion by etienne-bechara** [retypeapp/retype#131] [Ideas]
Currently, the only way I found to create a link that opens in a new tab, is to write pure HTML inside the markdown:

```
<a href="https://www.google.com" target="_blank">Google</a>
```

It would be nice if the button component supported that feature:

```
[!button target="blank" text="Google"](https://www.google.com)
```

**Comment by DannyBen** [comment]
Also discussed in #64

**Reply by geoffreymcgill** [reply]
@etienne-bechara Thanks for reminding us. We are definitely going to enable setting the `target` on hyperlinks and other Retype components, such as Button.

I'm looking into it today.

**Reply by geoffreymcgill** [reply]
Created an official Issue for this feature request, see #132.

**Reply by etienne-bechara** [reply]
Wow that was very fast! Thanks for looking into it!

**Reply by geoffreymcgill** [reply]
> Wow that was very fast! Thanks for looking into it!

Your timing was perfect.

**Reply by geoffreymcgill** [reply]
Just to tie this thread off, here are the docs for the new `target` property on Button and Badge:

https://retype.com/components/button/#target

Hope this helps.
